test_resources/ldap.ldif

```diff
@@ -40,10 +40,10 @@ objectClass: person
 objectClass: organizationalPerson
 objectClass: inetOrgPerson
 cn: Fred Taylor
-sN: Taylor
```

Comment:
I believe these were intentional, to test that attributes are case-insensitive.

It seems that LDIF supports comments, so maybe it would be worth leaving a comment here about that.